From: Gordon Ball <gordon@chronitis.net>
Date: Mon, 17 Jan 2022 15:10:06 +0000
Subject: pager: drop resizable

Until jquery-ui can be fixed, this provides a quick-and-dirty solution
for stopping the page load failing at this point, at the cost of the
element not being resizable by mouse.
---
 notebook/static/notebook/js/pager.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- jupyter-notebook.orig/notebook/static/notebook/js/pager.js
+++ jupyter-notebook/notebook/static/notebook/js/pager.js
@@ -23,7 +23,7 @@
         this.pager_element = $(pager_selector);
         this.pager_button_area = $('#pager-button-area');
         this._default_end_space = 100;
-        this.pager_element.resizable({handles: 'n', resize: $.proxy(this._resize, this)});
+        //this.pager_element.resizable({handles: 'n', resize: $.proxy(this._resize, this)});
         this.expanded = false;
         this.create_button_area();
         this.bind_events();
